PermissionsUtilsUI/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="colorPrimaryGreen">#008000</color>
```

Comment:
Move in Common UI and see with David the color name

Reply:
Done